content/practices/landmark-regions/landmark-regions-practice.html

```diff
@@ -252,33 +252,33 @@ <h4>HTML Techniques</h4>
               </li>
             </ul>
 
-            <h4>ARIA Technique</h4>
+            <h4>ARIAテクニック</h4>
 
-            <p>If the HTML <code>footer</code> element technique is not being used, a <code>role=&quot;contentinfo&quot;</code> attribute should be used to define a <code>contentinfo</code> landmark.</p>
+            <p>HTMLの<code>footer</code>要素のテクニックが使用されていない場合、<code>role=&quot;contentinfo&quot;</code>属性を使用して<code>contentinfo</code>ランドマークを定義します。</p>
           </section>
 
           <section>
-            <h4>Examples</h4>
-            <p><a href="../../patterns/landmarks/examples/contentinfo.html">Contentinfo Landmark Example</a></p>
+            <h4>例</h4>
+            <p><a href="../../patterns/landmarks/examples/contentinfo.html">補完ランドマークの例</a></p>
           </section>
         </section>
 
         <section id="aria_lh_form">
-          <h3 class="widget-name">Form</h3>
+          <h3 class="widget-name">フォーム</h3>
 
-          <p>A <a class="role-reference" href="#form"><code>form</code></a> landmark identifies a region that contains a collection of items and objects that, as a whole, combine to create a form when no other named landmark is appropriate (e.g. main or search).</p>
+          <p><a class="role-reference" href="#form"><code>form</code></a>ランドマークは、アイテムとオブジェクトの集合体を含む領域を表し、他の名前付きランドマーク（例：メインまたは検索）が適切でない場合、全体としてフォームを構成します。</p>
```

Comment:
```suggestion
          <p><a class="role-reference" href="#form"><code>form</code></a>ランドマークは、他の名前付きランドマーク（例：main、search）が適切でない場合、全体としてフォームを構成する、アイテムとオブジェクトの集合体を含む領域を特定します。</p>
```

Reply:
「領域」にかかる長めの修飾節が2つあり、やや認知負荷が高いかなーという印象を受けましたがいかがでしょうか？

2文に分けてもいいものでしょうかね・・

```suggestion
          <p><a class="role-reference" href="#form"><code>form</code></a>ランドマークは、アイテムとオブジェクトの集合体を含む領域を特定します。このランドマークは、他の名前付きランドマーク（例：main、search）が適切でない場合、全体としてフォームを構成します。</p>
```

Reply:
2文に分けるのがよいと思います！